worker/games.py

```diff
@@ -1080,32 +1080,41 @@ def launch_cutechess(
         w_params = req["w_params"]
         b_params = req["b_params"]
 
+        def generate_tune_options(params):
```

Comment:
a) this doesn't work with the branch from me because of the whitespaces after the commas
b) I guess it can be written more "pythonic" 
```python
options = ["{},{}".format(param["name"], math.floor(param["value"] + random.uniform(0, 1))) for param in params]
    
return ",".join(options)
```